src/sentry/analytics/events/comment_created.py

```diff
@@ -0,0 +1,16 @@
+from sentry import analytics
+
+
+class CommentCreatedEvent(analytics.Event):
```

Comment:
Since these are identical I think it'd make sense to use inheritance
```py
class CommentEvent(analytics.Event, abc.ABC):
    attributes = (
        ...
    )

class CommentCreatedEvent(CommentEvent):
    type = "comment.created"
```